tests/acceptance/features/coreApiVersions/fileVersions.feature

```diff
@@ -20,18 +20,12 @@ Feature: dav-versions
       | spaces           |
 
 
-  Scenario Outline: upload file and no version is available using various chunking methods (except new chunking)
-    Given using <dav-path-version> DAV path
-    When user "Alice" uploads file "filesForUpload/davtest.txt" to filenames based on "/davtest.txt" with all mechanisms except new chunking using the WebDAV API
```

Comment:
how about keeping the old scenario but with a modified step:
```feature
  Scenario Outline: upload file and no version is available using various chunking methods (except new chunking)
    Given using <dav-path-version> DAV path
    When user "Alice" uploads file "filesForUpload/davtest.txt" to "davtest.txt" with and without chunking using the WebDAV API
    Then the version folder of file "davtest.txt" for user "Alice" should contain "0" elements
    Examples:
      | dav-path-version |
      | old              |
      | new              |
      | spaces           |
```
you will need to fix the test code in method `uploadWithAllMechanisms`